mars/services/task/execution/api.py

```diff
@@ -89,3 +102,28 @@ def get_stage_processors(self):
 
 def register_executor_cls(executor_cls: Type[TaskExecutor]):
     _name_to_task_executor_cls[executor_cls.name] = executor_cls
+
+
+class Fetcher:
+    name = None
```

Comment:
Add doc for fetcher. ANd does this ,mean w need top create Fetcher object for every chunks?

Reply:
Doc added.

Reply:
The fetcher implementation has been refined.